presto-native-execution/src/test/java/com/facebook/presto/spark/PrestoSparkNativeQueryRunnerUtils.java

```diff
@@ -220,7 +221,7 @@ public static synchronized Path getBaseDataPath()
             }
         }
         else {
-            dataDirectory = Optional.of(Paths.get(dataDirectoryStr));
+            dataDirectory = Optional.of(NATIVE_WORKER_PARAMETERS.dataDirectory);
```

Comment:
I think there is another reference to PRESTO_SERVER on line 97 that should be replaced

Reply:
thanks, fixed